mathics/builtin/specialfns/gamma.py

```diff
@@ -332,7 +332,6 @@ class Gamma(MPMathMultiFunction):
 
     rules = {
         "Gamma[z_, x0_, x1_]": "Gamma[z, x0] - Gamma[z, x1]",
```

Comment:
In WMA, this rule only applies if the three arguments are numeric, and at least one is a Real number

Reply:
I imagine then that this kind of check would be better done inside an evaluation method. 

If you are up for adding, feel free to do. But if not, let's leave for another PR.

Reply:
OK, let's postpone for another PR.